### PR TITLE
🩹 [Tasks] Fix depth-estimation spec

### PR DESCRIPTION
**Pull request by SBrandeis**

### CODE REVIEW

File: packages/tasks/src/tasks/depth-estimation/inference.ts [hunk 1]
```diff
@@ -17,19 +15,21 @@ export interface DepthEstimationInput {
...
+export interface DepthEstimationOutput {
+	/**
+	 * The predicted depth as an image
+	 */
+	depth?: unknown;
 	/**
-	 * When specified, limits the output to the top K most probable classes.
+	 * The predicted depth as a tensor
 	 */
-	topK?: number;
```
Comment (osanseviero) [inline packages/tasks/src/tasks/depth-estimation/inference.ts:33]: Shouldn't we keep `topk` in parameters?

**Review by osanseviero — APPROVED** [review]
👍 LGTM!

File: packages/tasks/src/tasks/depth-estimation/inference.ts (hunk 1 above)
Comment (SBrandeis) [reply packages/tasks/src/tasks/depth-estimation/inference.ts:33]: It has been removed from the docstring in https://github.com/huggingface/transformers/pull/28733/files so I'm not sure

Ping @ydshieh who implemented the change in transformers

Comment (osanseviero) [reply packages/tasks/src/tasks/depth-estimation/inference.ts:33]: Ah yes! You're right hehe

**Pull request merged**